content/posts/newsletter/2019-09-03.md

```diff
@@ -0,0 +1,188 @@
+
+Please [submit your XMPP/Jabber articles, tutorials or blog posts](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter) on our wiki.
+
+<form style="padding: 10px; text-align:center; margin-bottom: 30px;"
+      action="https://tinyletter.com/xmpp" method="post" target="popupwindow"
+      onsubmit="window.open('https://tinyletter.com/xmpp', 'popupwindow',
+      'scrollbars=yes,width=800,height=600');return true">
+<p><label for="tlemail">Subscribe to receive the next edition in your inbox</label></p>
+<p><input type="text" placeholder="Email address" name="email" id="tlemail" /></p>
+<input type="hidden" value="1" name="embed"/>
+<input type="submit" style="padding: 10px; border-radius: 5%" value="Subscribe" />
+</form>
+
+
+## Articles
+
+Georg Lukas posted an article about the [10 years of yaxim](https://yaxim.org/blog/2019/08/23/happy-birthday/)!
+
+Marek Foss of ProcessOne, details the story on how they have [deployed an XMPP geocluster](https://blog.process-one.net/uniting-global-football-fans-with-an-xmpp-geocluster/) for the FIFA World Cup in Brazil.
+
+Cyril Brulebois wrote a short piece on [Sending HTML messages with Net::XMPP](https://mraw.org/blog/2019/08/17/Sending_HTML_messages_with_Net_XMPP/) (Perl).
+
+## Events
+
+Maxime “pep.” Buquet reports on the [XMPP sprint in Lyon](https://bouah.net/2019/07/new-sprint-new-goodies/) in July held at Wisolv's offices: DOAP - Description Of A Project, Reactions, Occupant-id, and more.
+
+The next [XMPP sprint in Stockholm](https://wiki.xmpp.org/web/Sprints/2019_September_Stockholm) will be held on September 28th and 29th, in the Nacka Municipality.
+
+A few [XMPP hackers met in Ziegeleipark, Mildenberg, Germany](https://wiki.xmpp.org/web/Camp2019) in August, for the Chaos Communication Camp 2019, an open-air hacker camp and party that takes place every four years, organized by the Chaos Computer Club (CCC).
+
+The regular meetups in Bavaria is held every third Monday every month. Next will [take place 2019-09-16](https://wiki.xmpp.org/web/Meetups/Bavaria).
+
+If you organise an event in your city, please [submit your meetups on the wiki](https://wiki.xmpp.org/web/Meetups).
+
+## Software releases
+
+### Servers
+
+[Prometheus XMPP Blackbox Exporter](https://github.com/horazont/prometheus-xmpp-blackbox-exporter) (Apache license 2.0) allows you to probe XMPP services and export metrics from the probes to Prometheus.
+
+The igniterealtime community has announced multiple releases:
+
+* [Openfire 4.4.1](https://discourse.igniterealtime.org/t/openfire-4-4-1-release/85857)
+* [Monitoring Service 1.8.0 and inVerse 4.2.0.1 plugins](https://discourse.igniterealtime.org/t/monitoring-service-1-8-0-and-inverse-4-2-0-1-plugins-released/85601)
+* [Smack 4.4.0-alpha2](https://discourse.igniterealtime.org/t/smack-4-4-0-alpha2-released/85670)
```

Comment:
Smack is a library. This release should probably not be listed under 'servers'.

Reply:
Done, thx.